components/activity_and_insights/insights/time_frame_dropdown/time_frame_dropdown.tsx

```diff
@@ -35,6 +37,10 @@ const TimeFrameDropdown = (props: Props) => {
             ...provided,
             cursor: 'pointer',
         }),
+        menuPortal: (provided: React.CSSProperties) => ({
+            ...provided,
+            zIndex: 100000,
```

Comment:
Does it need to be this high?

Reply:
I've reduced it to 1100, seems to be the lowest I can use on the modal